src/components/Integrations/Shopify/Shopify.js

```diff
@@ -24,23 +24,18 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     }
   `;
 
-  const Breadcrum = ({ className }) => (
-    <span className={className}>
-      <a className="main-section" href={_('common.control_panel_advanced_pref_url')}>
-        {_('common.control_panel')}
-      </a>{' '}
-      <span>&lt;</span>{' '}
-      <span className="secondary-section">{_('common.advanced_preferences')}</span>
-    </span>
+  const Breadcrumb = ({ className }) => (
```

Comment:
I don't think you are using the className parameter now, can you remove it please?